mongodb-replica-set-centos/azuredeploy.json

```diff
@@ -88,7 +92,13 @@
         "Standard_D4",
         "Standard_D2_v2",
         "Standard_D3_v2",
-        "Standard_D4_v2"
+        "Standard_D4_v2",
```

Comment:
let's just remove allowedSizes from these parameters, so we don't have to continue to maintain/update them